source/application/PGM/PGMAppController.cpp

```diff
@@ -87,7 +88,9 @@ void PGMAppController::setupScanConfigurations()
 
 void PGMAppController::setupUserConfiguration()
 {
-
+    PGMSlitControlView *slitView = new PGMSlitControlView();
```

Comment:
This code looks like its setting up the slits control view but it's in `setupUserConfiguration()`. Is this intentional, or did you mean to stick it in with other interface setup stuff?